rzup/rzup-init.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+# Copyright 2024 RISC Zero, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+# Example URLs:
+# for a 64-bit Linux system (x86_64):
+# BASE_URL/rzup-x86_64-linux
+# for a 64-bit macOS system (arm64):
+# BASE_URL/rzup-arm64-darwin
+
+# NOTE: Temporary release URL
+RZUP_BINARY_UPDATE_ROOT="${RZUP_BINARY_UPDATE_ROOT:-https://github.com/hmrtn/asset-test/releases/download/test1/}"
```

Comment:
This is a temporary URL as a placeholder for testing purpose only. Will be changed once we have the CI pipeline in place

Reply:
Seems like we can decide on the official URL now and just manually update it to get this landed.